src/coreclr/interop/comwrappers.cpp

```diff
@@ -666,6 +669,7 @@ HRESULT ManagedObjectWrapper::QueryInterface(
 
                 default:
                     _ASSERTE(false && "Unknown result value");
+                    [[fallthrough]];
```

Comment:
Forgot we only support C++14. Just comment out this line and we will uncomment it when we enable C++17.
```suggestion
                    // [[fallthrough]];
```

Reply:
Commenting it generates build errors : 

/runtime/src/coreclr/interop/comwrappers.cpp:673:17: error: unannotated fall-through between switch labels [-Werror,-Wimplicit-fallthrough]
                  case TryInvokeICustomQueryInterfaceResult::FailedToInvoke:
                  ^
  [ 14%] Built target System.IO.Ports.Native-Static
  [ 14%] Built target mdruntimerw-dbi
  [ 14%] Built target corerun
  [ 16%] Built target jitinterface_x64
  [ 16%] Built target coreclrtraceptprovider
  /runtime/src/coreclr/interop/comwrappers.cpp:673:17: note: insert '__fallthrough;' to silence this warning
                  case TryInvokeICustomQueryInterfaceResult::FailedToInvoke:
                  ^
                  __fallthrough;
  /runtime/src/coreclr/interop/comwrappers.cpp:673:17: note: insert 'break;' to avoid fall-through
                  case TryInvokeICustomQueryInterfaceResult::FailedToInvoke:
                  ^
                  break;
  [ 17%] Built target mdcompiler-dbi
  1 error generated.

Reply:
I believe somewhere in the repo we have a FALLTHROUGH macro. Maybe we can reuse that here?

Reply:
Yes, found it